dds_web/templates/technical_overview.html

```diff
@@ -0,0 +1,34 @@
+{% extends "base.html" %}
+
+{% block body %}
+<html>
+    <div class="container p-0" style="margin-top: 30px; font-size: medium;">
+        <h2>Technical Overview of the Data Delivery System</h2>
+        <p>The technical overview provides an overview of how the DDS works and is meant to be used. The document can be found on Confluence or downloaded as a PDF.</p>
+        <p class="lead"><a class="btn btn-info" href="https://scilifelab.atlassian.net/wiki/spaces/deliveryportal/pages/2201944070/2022+June+-+Technical+Overview" target="_blank">Read on Confluence</a>&nbsp;<a class="btn btn-info" href="https://scilifelab.atlassian.net/wiki/download/attachments/2201944070/2022-June_Technical-Overview.pdf?api=v2">Download the PDF</a>&nbsp;</p>
+        <br>
+    <footer class="footer">
+        <div class="container-fluid">
+            <div class="row text-muted">
+                <div class="col-md-10">
+                    <p>The Data Delivery System is developed and maintained by the <a href="https://scilifelab.se/data">SciLifeLab Data Centre</a>. With any questions please email datacentre@scilifelab.se.</p>
+                </div>
+            </div>
+        </div>
+    </footer>
+    </div>
+    
+        <!-- Cookie alert -->
+        <div class="d-flex justify-content-center cookiealert" role="alert">
+            <span style="display: flex;align-items: center;">We use cookies to improve your experience of the website. By continuing you accept that we use cookies.</span>
+        </div>
+    
+        <!-- JS code for the cookie alert-->
+        <script src="/static/dist/js/cookiealert.js"></script>
+        <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js"></script>
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/js/bootstrap.min.js"></script>
+        
+        <script src="/static/dist/js/app.js"></script>
```

Comment:
Can probably skip this part, mostly important for the policy part since they check that box when registering
```suggestion
```